lib/OperatingSystems/DefaultOs.php

```diff
@@ -215,6 +215,24 @@ public function getDiskInfo(): array {
 		return $data;
 	}
 
+	public function getThermalZones(): array {
+		$thermal_zones = glob('/sys/class/thermal/thermal_zone*');
```

Comment:
`glob()` might return `false`, causing PHP errors in the iteration below

Reply:
```suggestion
		$thermal_zones = glob('/sys/class/thermal/thermal_zone*') ?: [];
```
Same fix as the other instance by @PhrozenByte should be enough

Reply:
@come-nc Applied. Thank you for fixing it.